shared/src/commonMain/kotlin/com/bunbeauty/shared/presentation/menu/MenuViewModel.kt

```diff
@@ -168,7 +168,26 @@ class MenuViewModel(
             ),
         )
         sharedScope.launch {
-            addCartProductUseCase(menuProductUuid)
+
+            //для оптимизации заранее мапу отдельно хранить?
```

Comment:
Let's pass `hasAdditions` on click or the entire `MenuProductItem` to avoid `filterIsInstance` and `find` for optimization